test/Driver/pipe_round_robin.swift.gyb

```diff
@@ -0,0 +1,48 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/foo)
```

Comment:
Probably better to use the name of the test instead of "foo" here in case anyone ever needs to back trace.

Reply:
Ok.

Reply:
`%t` already includes the name of the test. I'd still be inclined to give it a stronger name, though.

(Also, no need to do `%t` first; `%empty-directory` is a form of `mkdir -p`.)

Reply:
fixed